README.md

```diff
@@ -126,6 +126,14 @@ python fizz_buzz.py
 ```
 
 ### Chat
+You can ask followup queries to the current chat using the `--refine` option:
+```shell
+sgpt 'Which letter comes after K in the alphabet?'
+# -> The letter that comes after K in the alphabet is L.
+sgpt -r "And the next?'
+# -> The letter that comes after L in the alphabet is M.
+```
+You can also maintain multiple chat sessions in parallel.
```

Comment:
```suggestion
You can also maintain multiple chat sessions in parallel by naming your chats with the --chat argument.
```

Reply:
The use of `--chat` is actually documented in the following sentence, but I can use your suggestion to make it flow better. Thanks!